src/views/Home/components/Schedule/index.view.tsx

```diff
@@ -1,8 +1,101 @@
-import React from "react";
+/* eslint-disable @typescript-eslint/no-var-requires */
+import { useState } from "react";
+import * as React from "react";
 import "./schedule.scss";
 
+const scheduleItems = [
+  {
+    date: "Friday, January 15, 2021",
+    events: [
+      { event: "Event Begins", time: "5:00 pm" },
+      { event: "Opening Ceremony", time: "7:00 pm" },
+      { event: "Hacking Begins!", time: "9:00 pm" },
+      { event: "Ongoing Workshops", time: "10:00 pm" },
+    ],
+  },
+  {
+    date: "Saturday, January 16, 2021",
+    events: [{ event: "Ongoing Workshops", time: "10:30 am" }],
+  },
+  {
+    date: "Sunday, January 17, 2021",
+    events: [
+      { event: "Project Submissions", time: "10:00 am" },
+      { event: "Judging", time: "11:30 am" },
+      { event: "Closing Ceremony", time: "2:00 pm" },
+      { event: "Event Ends", time: "4:00 pm" },
+    ],
+  },
+];
 const ScheduleComponent: React.FC = () => {
-  return <div className="Schedule">Hello from ScheduleComponent</div>;
+  const changeDay = (day: number) => {
+    const prevActive = document.getElementById(`day${selectedDay + 1}`);
+    if (prevActive) {
+      prevActive.classList.remove("Schedule__tab--variant");
+      prevActive.style.zIndex = "-3";
+    }
+    const currActive = document.getElementById(`day${day}`);
+    if (currActive) {
+      currActive.classList.add("Schedule__tab--variant");
+      currActive.style.zIndex = "0";
+      setSelectedDay(day - 1);
+    }
+  };
```

Comment:
If this is what we are using to style and change class names, take a look at the [`classnames`](https://www.npmjs.com/package/classnames) package and how it's used here: 
https://github.com/CruzHacks/cruzhacks-2021-website/blob/482cd786794a84bc5c79d8824a45ef87ff7c2749/src/views/Hero/index.view.tsx#L30-L32

In turn, having this logic here would allow you to do all the styling in the SCSS regardless of the conditionals in the TS code.